qbft/spectest/tests/controller/decided/current_instance.go

```diff
@@ -74,16 +74,6 @@ func CurrentInstance() *tests.ControllerSpecTest {
 							Data:       testingutils.CommitDataBytes([]byte{1, 2, 3, 4}),
 						}),
 				},
-				SavedDecided: testingutils.MultiSignQBFTMsg(
```

Comment:
don't we want to check that these messages were at least sent to broadcast?

Reply:
in this particular test we don't broadcast anything